### ISSUE TITLE
Pipes, tubes, and cables suddenly stop functioning (especially with multiblock structures)

### ISSUE THREAD

**Issue by thelittleskeleton** [mekanism/Mekanism#7778]
### Issue description

I've noticed recently that a lot of my machines haven't been sending or recieving the resources they need to function; when checking their inputs/outputs they are configured correctly for what they should be sending or recieving but the connections between machines don't transport the resources. I have adequate power for everything I'm using. This issue has resulted in a total meltdown of my fission reactor, which leveled half my base and irradiated everything around it because steam and water were not being transferred between the turbine and reactor. I have tried using the Configurator to change the connection types, and even tried breaking/replacing the structures, but nothing worked. The solution I found for my small, one-block machines is to simply recraft them and place them again, which fixes this issue, but it's very resource-intensive and I would really like to know how to solve this.
Below is a list of all my mods:
![image](https://user-images.githubusercontent.com/37987997/233121166-1a082590-bcd8-47a3-a541-f962c98f3e4e.png)
I feel like it might be an issue with chunk loading; despite this I have unpowered Teleporter blocks with the Anchor Upgrade installed in every chunk where the fission reactor & related systems are.

### Steps to reproduce

Unsure how to reproduce (it happened randomly during regular gameplay)

### Minecraft version

1.19.2 (Latest)

### Forge version

43.2.0

### Mekanism version

10.3.8 (Latest)

**Comment by thelittleskeleton** [comment]
Sure, here
[2023-04-19-2.log.gz](https://github.com/mekanism/Mekanism/files/11275180/2023-04-19-2.log.gz)
I broke and replaced the coolant output line for the reactor and it works now, but this is a weird issue that's been bugging me a lot. 
It also could be a mistake I made with the Configurator where I accidentally changed a connection somewhere and didn't notice.

**Comment by MrGreenBud** [comment]
Same problem here. Except It breaks completely randomly, have tried replicating it by going into different dimensions, shutting down the server and etc. It will just stop sending/receiving items and stop sending/receiving energy.
[latest.log](https://github.com/mekanism/Mekanism/files/11409865/latest.log)

**Comment by MrGreenBud** [comment]
Also seems to happen with the Ultimate Energy Cube as well. It will just stop receiving energy/sending energy.

**Comment by michaelortnerit** [comment]
Same issue here, i playing TNP Limitless 6, input works to the enriching factory and with a pipe to the smelting factory too, but if i want to eject the ingots to a chest it doesn't work, it stucks in the machine

![image](https://github.com/mekanism/Mekanism/assets/135371107/aa1411c4-77d6-4047-adf7-1822ac9d7a7e)

**Comment by NeonHaiiroookami** [comment]
I agree and I don't know why it kept happening like this... But I'm using Mekanism 10.3.9 and Forge 43.2.21

**Comment by sunfirecats** [comment]
I have the same issue has there been any updates on how to fix it

**Comment by Aceplante** [comment]
pipez seem to stop transferring to dynamic tanks when the chunks are reloaded
